editions/tw5.com/tiddlers/howtos/How to hide the authors and other fields with CSS.tid

```diff
@@ -0,0 +1,32 @@
+created: 20180309232911422
+modified: 20180310035732641
+tags: Learning
+title: How to hide the author's and other fields with CSS
+type: text/vnd.tiddlywiki
+
+Sometimes you might want to save some screen space by hiding away the author's name in all tiddlers. Here's a quick way to do it:
+
+* Create a new tiddler with any title you want.
+* Give the tiddler the tag `$:/tags/Stylesheet`
+* In the text field of the tiddler put:
+
+```
+a.tc-tiddlylink.tc-tiddlylink-missing {display:none;}
```

Comment:
This looks like it's actually hiding links to missing tiddlers, which would include more than just the username.